### ISSUE TITLE
Electron crashes/hangs on attempting to use Application Menu with hidden webview not yet activated

### ISSUE THREAD

**Issue by TimNZ** [electron/electron#6687]
* Electron version: 1.3.1
* Operating system: OS X 10.11.5

Electron will crash/hang if attempting to use the Application Menu when a webview has not been made visible, in this case contained within a div with 'style="display: none"'

Run sample app to replicate - 'electron main.js'

Manually clear style on container div to cause webview to display, no longer crashes on use of app menu.

Does not error in Windows

[browser.zip](https://github.com/electron/electron/files/395181/browser.zip)

**Comment by kevinsawicki** [comment]
Which menu item are you selecting for it to crash? I'm having trouble reproducing locally.

**Comment by TimNZ** [comment]
Updated webview.html below to make it easier to toggle

Below are two gif showing the difference.  Couldn't capture the menu bar using Giphy.
All menu items trigger a crash when the webview is display: none and has not been made visible previously.  In both I simply either using a menu item keyboard accelerator or select directly from menu.  The top level menu does not trigger a crash

I'm creating a fresh OS X VM now to see if it is environmental.
Have removed global electron-prebuilt, deleted ~/Library/Application Support/Electron dir, and re-installed.

Crash report: [CrashReport.txt](https://github.com/electron/electron/files/398218/CrashReport.txt)

Thread 0 crashes

Crashes:

![crash](https://cloud.githubusercontent.com/assets/1051701/17346588/8d6a4d42-595f-11e6-946a-93e18817b500.gif)

No Crash:


![nocrash](https://cloud.githubusercontent.com/assets/1051701/17346587/8d699eba-595f-11e6-8a3a-5a1cb2ce5251.gif)

```
<html>
    <head>
        <script>if (typeof module === 'object') {window.module = module; module = undefined;}</script>
        <script src="https://code.jquery.com/jquery-3.1.0.js"></script>
        <script>if (window.module) module = window.module;</script>
    </head>
    <body>
        <button onclick="toggleDisplay()">Toggle Display</button>
        <div style="display: none" id="webview">
        <webview src="https://inbox.google.com" partition="persist:inbox"></webview>
        </div>
        <script>
            function toggleDisplay()
            {
                $('#webview').toggle()
            }
        </script>

    </body>
</html>
```

**Comment by kevinsawicki** [comment]
Thanks for the crash report, it looks like a duplicate of https://github.com/electron/electron/issues/6642 which was fixed in #6644 and will be included in the next Electron release, 1.3.2, which should be out this week.

**Comment by TimNZ** [comment]
Cool thanks.  Sorry for the dupe.  

I thought I did an issue search for 'webview crash' beforehand and perhaps did not as I can see a few issues related to it.
